### PR TITLE
Christmas refactor

### PR DESCRIPTION
**Pull request by nipatiitti**
- Added new file structure
- Refactored christmas command to be year insensitive
- Added docker-compose
- Modified readme
- Updated the API

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+version: '2'
```
Comment (nipatiitti) [inline docker-compose.yml:1]: Can't believe we didn't have this before. Such a good thing

File: src/commands/christmas.ts [hunk 2]
```diff
@@ -1,90 +1,117 @@
+import axios from 'axios'
 import dayjs from 'dayjs'
+import isToday from 'dayjs/plugin/isToday'
+import FormData from 'form-data'
+import syncFs from 'fs'
+import fs from 'fs/promises'
 import { Commander } from '.'
-import _ from 'lodash'
```
Comment (nipatiitti) [inline src/commands/christmas.ts:3]: Get this trash outta here

File: src/commands/christmas.ts [hunk 3]
```diff
@@ -1,90 +1,117 @@
...
-            lastDate = today.date()
-          })
-          .catch(reason => {
-            console.log('Failed to send image')
-          })
-        */
+            const pre_res = await telegram.sendMessage(message.chat.id, 'Avataan luukkua!!')
+            const message_id = pre_res.message_id
+
+            try {
```
Comment (nipatiitti) [inline src/commands/christmas.ts:64]: Such a good code. This writer has to be a god. I really cant believe it

**Review by nipatiitti — COMMENTED** [review]
This is some good shit.

**Review by HammerCar — COMMENTED** [review]
Mint

File: docker-compose.yml (hunk 1 above)
Comment (HammerCar) [reply docker-compose.yml:1]: Nice

File: src/commands/christmas.ts (hunk 2 above)
Comment (HammerCar) [reply src/commands/christmas.ts:3]: It never should have existed

File: src/commands/christmas.ts (hunk 3 above)
Comment (HammerCar) [reply src/commands/christmas.ts:64]: Yeah sure

**Pull request merged**